rholang/README.md

```diff
@@ -25,7 +25,7 @@ Currently we have a working interpreter for the language. It should be considere
 			exec /usr/bin/java java_cup.Main "$@"
 			```
     * JFlex - install using apt 
-    * BNFC - must be built from [git](https://github.com/BNFC/bnfc) b0252e5f666ed67a65b6e986748eccbfe802bc17 or later
+    * BNFC - MUST be built from [git](https://github.com/BNFC/bnfc) b0252e5f666ed67a65b6e986748eccbfe802bc17 or later (even though BNFC found in your OS repository might match the version on github it may not work -- we higly recommend building from the source). If you use `cabal install` you will need to add your BNFC binary to the PATH.
```

Comment:
I like the "MUST", but the part about OS repository seems a bit redundant.

Also should we provide step by step instructions for cloning and running cabal?

If we do, we should suggest that users use `cabal sandbox` or `cabal new-install` instead of `cabal install`.  I don't want to be responsible for leading unsuspecting users down the road towards the infamous "Cabal Hell".

Reply:
I agree it's redundant. 

About the cabal part -- I see what you mean and I agree that this might prevent few people from getting into trouble. On the other hand, if I were using cabal on daily basis I think that I would be aware of the **Cabal Hell**. I don't have the knowledge on that, if you want to write this down please feel free. Although I hope it's not gonna take a lot of your time.

Reply:
I think it's safe to assume that not everyone attempting to build our project will be familiar with the vagaries of using `cabal`.  I will open a separate PR with those instructions (it's a pretty quick change).